pysat/tests/classes/cls_instrument_access.py

```diff
@@ -179,7 +222,17 @@ def test_basic_instrument_load_no_input(self):
                              [('fname', 'have multi_file_day and load by file'),
                               (None, 'is not supported with multi_file_day')])
     def test_instrument_load_errors_with_multifile(self, load_in, verr):
-        """Ensure load calls raises ValueError with `multi_file_day` as True."""
+        """Ensure load calls raises ValueError with `multi_file_day` as True.
```

Comment:
```suggestion
        """Ensure load by filename raises ValueError with `multi_file_day` as True.
```

This actually could be a potential issue for users. Originally, `multi_file_day` was an input keyword argument but that was removed. This means for instruments where we set `multi_file_day=True` (DMSP, ICON IVM) users can't load by file anymore....

Potential solutions:
 - bring back argument
 - Update documentation to reflect that loading by file for some instruments requires setting an instrument attribute to False.

Out of scope for this pull. Making comments here so I don't forget when issue made.

Reply:
This is probably a bug that needs to be fixed. I can see reasons to want the option to load by file.